frontend/src/lib/components/SelectGradientOverflow.tsx

```diff
@@ -100,7 +100,7 @@ export function SelectGradientOverflow({
     }
     document.addEventListener('click', outsideClickListener)
     return (
-        <div ref={containerRef}>
+        <div ref={containerRef} style={{ width: '100%' }}>
```

Comment:
This is the relevant change - the other diffs look large but it's just the formatting that changed after removing the fragment